drivers/google/google.go

```diff
@@ -260,20 +271,21 @@ func (d *Driver) Remove() error {
 	return c.deleteDisk()
 }
 
-// Restart deletes and recreates the GCE instance, keeping the disk.
 func (d *Driver) Restart() error {
+	return nil
+}
+
+// Kill deletes the GCE instance, but keeps the disk.
```

Comment:
I know it's not your fault but, I don't think this is the intention of `Kill`.  It's meant to be a "hard stop".